docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -269,7 +273,26 @@ For example, the following command runs tests from the specified file and forces
 testcafe ie tests/sample-fixture.js -e
 ```
 
-### -t \<name\>,  --test \<name\>
+### -c \<factor\>, --concurrency \<factor\>
+
+Runs tests concurrently. The `factor` parameter specifies the number of browser instances
```

Comment:
I agree with @churkin.

The first two sentences are:
```js
Runs tests concurrently. ... the number of browser instances that will be invoked at the same time to run tests concurrently.
```
It's actual not for `concurrently mode` only. When we run just `testcafe chrome,firefox test.js` we run tests **concurrently** in two browsers. So, we should chose a wording that shows the difference clearly.

I think we can add a new section somewhere in the documentation that describes this mode in details. And we can post a link to it from the CLI API and Programming API sections.